connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -187,7 +219,11 @@ public class ConnectorConfig extends AbstractConfig {
     public static final String CONNECTOR_CLIENT_ADMIN_OVERRIDES_PREFIX = "admin.override.";
     public static final String PREDICATES_PREFIX = "predicates.";
 
-    private final EnrichedConnectorConfig enrichedConfig;
+    private static final PluginsRecommenders EMPTY_RECOMMENDER = new PluginsRecommenders();
+    private static final ConverterDefaults CONVERTER_DEFAULTS = new ConverterDefaults();
```

Comment:
Since you're going to use ConverterDefaults in a static field like this, you should make it immutable, and maybe even take advantage of the new Java Record type.